src/test/java/io/bf2/kafka/authorizer/CustomAclAuthorizerTest.java

```diff
@@ -59,20 +62,20 @@ static void initialize() throws IOException {
 
     @BeforeEach
     void setup() {
-        this.delegate = Mockito.mock(kafka.security.authorizer.AclAuthorizer.class);
-
-        Mockito.when(this.delegate.authorize(Mockito.any(AuthorizableRequestContext.class), Mockito.anyList()))
-            .thenAnswer(invocation -> {
-                int count = invocation.getArgument(1, List.class).size();
-                List<AuthorizationResult> results = new ArrayList<>(count);
-                for (int i = 0; i < count; i++) {
-                    results.add(AuthorizationResult.DENIED);
-                }
-                return results;
-            });
-
-        Mockito.when(this.delegate.acls(Mockito.any(AclBindingFilter.class)))
-            .thenReturn(Collections.emptyList());
+        this.delegate = mock(kafka.security.authorizer.AclAuthorizer.class);
+
+        when(this.delegate.authorize(any(AuthorizableRequestContext.class), anyList()))
+                .thenAnswer(invocation -> {
```

Comment:
It is distracting when a PR that changes functionality also reformats code.

Reply:
Fixed